modules/json_content/src/Form/UserLoginForm.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Drupal\json_content\Form;
+
+use Drupal\user\Form\UserLoginForm as UserLoginFormBase;
```

Comment:
Can this import be changed to BaseUserLoginForm instead? Having 'Base' at the end implies an abstract base class.